src/uwtools/config/support.py

```diff
@@ -56,6 +56,15 @@ def format_to_config(fmt: str) -> Type:
     return cfgclass
 
 
+def from_od(d: Union[OrderedDict, Dict]) -> dict:
+    """
+    Returns a (nested) dict with content equivalent to the given (nested) OrdereDict.
+
+    :param d: A (possibly nested) OrderedDict.
+    """
+    return {key: from_od(val) if isinstance(val, dict) else val for key, val in d.items()}
```

Comment:
I needed this for a unit test, so factored it out of the `_represent_ordereddict()` method, below.